### DISCUSSION TITLE
Why can't control.limits acceleration be lower than 50?

### DISCUSSION THREAD

**Discussion by pqhf5kd** [pybricks/support#1359] [General]
I get an Invalid Argument when I try to set control.limits acceleration lower than 50. 
Is that a bug or is there a reason?

**Comment by laurensvalk** [comment]
At 50 deg/s^2, it would take 20 seconds to get to full speed (1000 deg/s), so it is presumed that anything slower is rarely needed.

Having a maximum on the acceleration time (around 20 seconds) simplifies a lot of the internal math, so we put a cap there.

If you really do need something slower than 20 seconds, timing is perhaps not so critical, so you could make your own function that increments in the speed in `run(speed)` as slowly as you like. Keep in mind that these motors aren't really designed to run at less than "15%" speed, or 150 deg/s, so it may not run very smoothly at such low speeds.
